src/gui.rs

```diff
@@ -36,56 +47,32 @@ fn create_icon() -> Result<Icon> {
 
 #[derive(Debug, Default)]
 struct State {
-    minecraft_pick_list: pick_list::State<MinecraftVersion>,
+    // MC version picker
     minecraft_versions: Vec<MinecraftVersion>,
     selected_minecraft_version: Option<MinecraftVersion>,
     show_snapshots: bool,
 
-    loader_pick_list: pick_list::State<LoaderVersion>,
+    // Quilt loader version picker
```

Comment:
Nitpick:
```suggestion
    // Quilt Loader version picker
```

Reply:
Hoho how did this get through.?
(I'm also _very_ particular about capitalisation of proper nouns)